staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -339,12 +359,6 @@ var (
 					[]string{"endpoints", "configmaps"},
 					[]string{flowcontrol.NamespaceEvery},
 					false),
-				resourceRule(
```

Comment:
why remove this?  We want them to use leases, right?

Reply:
same comment as https://github.com/kubernetes/kubernetes/pull/106725#discussion_r768877109

> leases are already covered by system-leader-election above, right? I think the intent is for this this to only back-fill configmaps and endpoints